### PR TITLE
doc: CCDIKSolver with generic SkinnedMesh

### PR DESCRIPTION
**Pull request by abernier**
> **Note** Merged<br> see [`CCDIKSolver` doc](https://threejs.org/docs/index.html?q=ccdik#examples/en/animations/CCDIKSolver)

Align the CCDIKSolver documentation with its description, which says it works with a generic SkinnedMesh (not only MMDLoader).

Adding an embed example, based on the SkinnedMesh one.

See:
- https://discourse.threejs.org/t/example-of-how-to-use-ccdiksolver-with-a-generic-skinnedmesh/9571/12
- https://codesandbox.io/s/romantic-rubin-n0df6?file=/src/index.js

### CODE REVIEW

**Comment by Mugen87** [comment]
https://raw.githack.com/abernier/three.js/ccdiksolver-doc-example/docs/index.html#examples/en/animations/CCDIKSolver

**Comment by Mugen87** [comment]
Looks good to me!

@takahirox Are you fine with this change?

File: docs/examples/en/animations/CCDIKSolver.html [hunk 1]
```diff
@@ -11,31 +11,94 @@ <h1>[name]</h1>
 
 		<p class="desc"> A solver for IK with <a href="https://sites.google.com/site/auraliusproject/ccd-algorithm"><em>CCD Algorithm</em></a>. <br /><br />
 		[name] solves Inverse Kinematics Problem with CCD Algorithm.
-		[name] is designed to work with [page:SkinnedMesh] loaded by [page:MMDLoader] but also can be used for generic [page:SkinnedMesh].
+		[name] is designed to work with [page:SkinnedMesh] but also can be used with [page:MMDLoader] or [page:GLTFLoader] skeleton.
 		</p>
 
+		<iframe id="scene" src="scenes/ccdiksolver-browser.html"></iframe>
```
Comment (Mugen87) [inline docs/examples/en/animations/CCDIKSolver.html:17]: On tiny thing: Can you also include the iFrame and the updated code example in the `zh` version?

Comment (abernier) [reply docs/examples/en/animations/CCDIKSolver.html:17]: Sorry what do you call `zh` version?

Comment (Mugen87) [reply docs/examples/en/animations/CCDIKSolver.html:17]: Check out: https://threejs.org/docs/index.html#examples/zh/animations/CCDIKSolver

(The page is just not translated, yet.)

File: docs/examples/en/animations/CCDIKSolver.html [hunk 2]
```diff
@@ -11,31 +11,94 @@ <h1>[name]</h1>
...
+		// Bones hierarchy:
+		//
+		//   root
+		//     ├── bone0
+		//     │    └── bone1
+		//     │          └── bone2
+		//     │                └── bone3
+		//     └── target
+		//
+		// Positionned as follow on the cylinder:
```
Comment (LeviPesin) [inline docs/examples/en/animations/CCDIKSolver.html:34]: Typo: "positioned", not "positionned".

File: docs/examples/en/animations/CCDIKSolver.html [hunk 3]
```diff
@@ -11,31 +11,94 @@ <h1>[name]</h1>
...
+
+		const iks = [
+			{
+				target: 5, // "target"
+				effector: 4, // "bone3"
+				links: [{ index: 3 }, { index: 2 }, { index: 1 }] // "bone2", "bone1", "bone0"
 			}
-		);
+		];
+		ikSolver = new CCDIKSolver(mesh, iks);
```
Comment (LeviPesin) [inline docs/examples/en/animations/CCDIKSolver.html:97]: Here (and possibly in the previous line) some lint spaces are missing.

File: docs/examples/en/animations/CCDIKSolver.html (hunk 1 above)
Comment (abernier) [reply docs/examples/en/animations/CCDIKSolver.html:17]: got it! I will add an updated version of https://github.com/mrdoob/three.js/blob/dev/docs/examples/zh/animations/CCDIKSolver.html 👍

File: docs/examples/en/animations/CCDIKSolver.html (hunk 3 above)
Comment (abernier) [reply docs/examples/en/animations/CCDIKSolver.html:97]: I will add them 👍 btw for convenience, is there any linter already setup in the project I can use?

Comment (Mugen87) [reply docs/examples/en/animations/CCDIKSolver.html:97]: Editors like VSCode or Atom should automatically detect the ESLint config of the repository.

**Comment by takahirox** [comment]
> @takahirox Are you fine with this change?

Yes, this change looks good. `CCDIKSolver` is originally made for MMD but also designed for generic use. And we have good document now. So mentioning that it works with a generic SkinnedMesh (not only MMDLoader) sounds good to me.

File: docs/examples/en/animations/CCDIKSolver.html (hunk 2 above)
Comment (abernier) [reply docs/examples/en/animations/CCDIKSolver.html]: fixed

File: docs/examples/en/animations/CCDIKSolver.html (hunk 1 above)
Comment (abernier) [reply docs/examples/en/animations/CCDIKSolver.html:17]: added in https://github.com/mrdoob/three.js/pull/23449/commits/298a6f0c4f07ededdd72c66c25740cd95243ae3e

File: docs/examples/en/animations/CCDIKSolver.html (hunk 3 above)
Comment (abernier) [reply docs/examples/en/animations/CCDIKSolver.html]: Btw, eslint was not detected automatically, maybe because it's a `.html` file here

**Review by Mugen87 — APPROVED** [review]

**Comment by mrdoob** [comment]
Thanks!

**Pull request merged**